src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -96,10 +99,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $io->comment('Re-run the command with a -vv option to see logs about consumed messages.');
         }
 
-        $receiver = $this->getReceiver();
+        $failureTransportName = $input->getOption('failure-transport');
+        if (null === $failureTransportName) {
+            $failureTransportName = $this->getGlobalFailureReceiverName();
```

Comment:
In this situation - where they are running the command without the `--failure-transport=` option, I think we need to advertise that there are other failure transports. So, if there ARE other failure transports, we would say (at the top of the command output):

```
> Lading messages from the <comment>global</comment> failure transport. 
> To use a different failure transport, pass`--failure-transport=`.
> Available failure transports are: foo_failure, bar_failure

There are 2 messages pending in the failure transport.
```

We would need to play around with the formatting... I was thinking maybe the `$io->comment()` functionality for the new message (the last line already exists, I was just showing where the new message would fit in).

Reply:
I did for the retry and show commands. The remove, doesn't make sense because id is required.